modules/rosa-aws-scp.adoc

```diff
@@ -1,16 +1,42 @@
 // Module included in the following assemblies:
 //
-// * rosa_getting_started/rosa-aws-prereqs.adoc
+// * rosa_architecture/rosa-sts-about-iam-resources.adoc
+//
+// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc
+//
 
 [id="rosa-minimum-scp_{context}"]
-== Minimum required service control policy (SCP)
+= Minimum required service control policy (SCP)
+
+Service control policies (SCP) are a type of organization policy that manages permissions within your organization. SCPs ensure that accounts within your organization stay within your defined access control guidelines. These polices are maintained in the AWS Organizations and control what services are available within the attached AWS accounts. SCP management is the responsibility of the customer.
```

Comment:
s/what services/which services  -- or --  the services that

Reply:
SCP management is your responsibility ?? The rest of the paragraph and the following are in the _your_ format. _the responsibility of the customer._ sounds awkward in this context. (Although it is probably the stronger way to state it.)